tests/js/common/shell/shell-collection-rocksdb-noncluster.js

```diff
@@ -408,102 +408,106 @@ function CollectionSuite () {
+            inital[i] = idx.figures;
+          }
+        });
+        for(let i = 0; i < 10000; i++) {
+          c.outEdges("c/v" + (i / 100));
+          c.inEdges("c/v" + (i / 100));
         }
-      });
-
-      // cache usage should be 0 after unload
-      c.unload(); // destorys cache
-      idxs = c.getIndexes(true); // loads collection
-      idxs.forEach(function(idx, i) {
-        if (idx.figures.cacheInUse) {
-          assertTrue(idx.figures.cacheSize > 0);
-          assertEqual(idx.figures.cacheUsage, 0);
+        idxs = c.getIndexes(true);
+        // cache was filled with same queries, hit rate must be higher
+        idxs.forEach(function(idx, i) {
+          if (idx.figures.cacheInUse) {
+            assertTrue(Math.abs(inital[i].cacheSize - idx.figures.cacheSize) < 1024);
+            assertTrue(idx.figures.cacheLifeTimeHitRate > 30, idx);
+          }
+        });
+
+        // cache usage should be 0 after unload
+        c.unload(); // destorys cache
+        idxs = c.getIndexes(true); // loads collection
+        idxs.forEach(function(idx, i) {
+          if (idx.figures.cacheInUse) {
+            assertTrue(idx.figures.cacheSize > 0);
+            assertEqual(idx.figures.cacheUsage, 0);
+          }
+        });
+
+        // lets do some reads
+        for(let i = 0; i < 10000; i++) {
+          c.outEdges("c/v" + (i / 100));
+          c.inEdges("c/v" + (i / 100));
         }
-      });
-
-      // lets do some reads
-      for(let i = 0; i < 10000; i++) {
-        c.outEdges("c/v" + (i / 100));
-        c.inEdges("c/v" + (i / 100));        
-      }
-      idxs = c.getIndexes(true);
-      // cache was empty, hit rate should be 0
-      idxs.forEach(function(idx) {
-        if (idx.figures.cacheInUse) {
-          assertTrue(idx.figures.cacheSize > 0, idx);
-          assertTrue(idx.figures.cacheUsage > 0, idx);
-          assertEqual(idx.figures.cacheLifeTimeHitRate, 0, idx);
+        idxs = c.getIndexes(true);
+        // cache was empty, hit rate should be 0
+        idxs.forEach(function(idx) {
+          if (idx.figures.cacheInUse) {
+            assertTrue(idx.figures.cacheSize > 0, idx);
+            assertTrue(idx.figures.cacheUsage > 0, idx);
+            assertEqual(idx.figures.cacheLifeTimeHitRate, 0, idx);
+          }
+        });
+
+        for(let i = 0; i < 10000; i++) {
+          c.outEdges("c/v" + (i / 100));
+          c.inEdges("c/v" + (i / 100));
         }
-      });
-
-      for(let i = 0; i < 10000; i++) {
-        c.outEdges("c/v" + (i / 100));
-        c.inEdges("c/v" + (i / 100));        
+        idxs = c.getIndexes(true);
+        // cache was partially filled same queries, lifetime hit rate
+        // should be about 50 %
+        idxs.forEach(function(idx) {
+          if (idx.figures.cacheInUse) {
+            assertTrue(idx.figures.cacheLifeTimeHitRate > 15, idx);
+          }
+        });
+      } catch (x) {
+        db._drop(x);
```

Comment:
should be `db._drop(cn);`, not `db._drop(x);`.
However, you use try..catch here and not try..finally?
That would be a lot better!